load-jetpack.php

```diff
@@ -0,0 +1,106 @@
+<?php
+/**
+ * Load all Jetpack files that do not get loaded via the autoloader.
+ *
+ * @package Jetpack
+ */
+
+/**
+ * Returns the location of Jetpack's lib directory. This filter is applied
+ * in require_lib().
+ *
+ * @since 4.0.2
+ *
+ * @return string Location of Jetpack library directory.
+ *
+ * @filter require_lib_dir
+ */
+function jetpack_require_lib_dir() {
+	return JETPACK__PLUGIN_DIR . '_inc/lib';
+}
+add_filter( 'jetpack_require_lib_dir', 'jetpack_require_lib_dir' );
+
+/**
+ * Checks if the code debug mode turned on, and returns false if it is. When Jetpack is in
+ * code debug mode, it shouldn't use minified assets. Note that this filter is not being used
+ * in every place where assets are enqueued. The filter is added at priority 9 to be overridden
+ * by any default priority filter that runs after it.
+ *
+ * @since 6.2.0
+ *
+ * @return boolean
+ *
+ * @filter jetpack_should_use_minified_assets
+ */
+function jetpack_should_use_minified_assets() {
+	if ( defined( 'SCRIPT_DEBUG' ) && SCRIPT_DEBUG ) {
+		return false;
+	}
+	return true;
+}
+add_filter( 'jetpack_should_use_minified_assets', 'jetpack_should_use_minified_assets', 9 );
+
+// @todo: Abstract out the admin functions, and only include them if is_admin()
+require_once JETPACK__PLUGIN_DIR . 'class.jetpack.php';
```

Comment:
do we break something if we put all the `require_once` at the top?

Reply:
I haven't tried, I followed how things were before they were moved to a separate file.